src/helpers/array.helper.ts

```diff
@@ -1,13 +1,13 @@
 export class ArrayHelper {
-  public static zip<T>(array1: T[], array2: T[]): [T, T][] {
-    let longer: T[];
-    let shorter: T[];
-    if (array1.length >= array2.length) {
-      [longer, shorter] = [array1, array2];
-    } else {
-      [longer, shorter] = [array2, array1];
+  public static zip<T1, T2>(
+    array1: (T1 | null)[],
+    array2: (T2 | null)[]
+  ): [T1 | null, T2 | null][] {
+    if (array1.length < array2.length) {
+      const count = array2.length - array1.length;
+      array1.fill(null, count);
```

Comment:
Taki `fill` nie zadziała.
![image](https://user-images.githubusercontent.com/21079319/99000632-e1683a80-2539-11eb-8fd0-3deaa63cf19b.png)

Nie wiem szczerze mówiąc dlaczego, ale tak to jest zrobione w JS.
Potrzeba spróbować czegoś innego.
Tu są podane różne opcje
https://stackoverflow.com/a/28599347/7516134